src/index.js

```diff
@@ -59,7 +59,7 @@ export function app(state, actions, view, container) {
     return node === null
       ? h(node, null)
```

Comment:
@frenzzy What do you think if we check in all the nodes if there's any null node ?